File: Bab 7/7.1. If.java
import java.util.Scanner;

public class Main{

    public static void main(String[] args){

        char nilai = '\0';

        System.out.println("masukkan karakter:");

        Scanner sc = new Scanner(System.in);

        try{

            nilai = sc.next().charAt(0);
        
        }catch(Exception e){

            if((nilai == 'A') || (nilai == 'B') || (nilai == 'C')){
                
                System.out.println("lulus");
            }
        }
    }
}

File: Bab 7/7.1. If.c
/*
 *  Name Program    : If
 *  Purpose         : Try Code from Logic Algorithm and Basic Programming Book chapter 7
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : C Language
 */

#include <stdio.h>

// Main function
int main(){

    // Variable instantiation
    char nilai;

    printf("Masukan karakter:\n");

    // Get input from console
    scanf("%c", &nilai);

    // Condition when nilai is equal with A, or B, or C
    if((nilai == 'A') || (nilai == 'B') || (nilai == 'C')){

        printf("lulus\n");
    }

    return 0;
}